client/app/components/project/project-editor-list-item.js

```diff
@@ -3,12 +3,14 @@ import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
 
 export default class ProjectEditorListComponent extends Component {
-  get disableDelete() {
-    return this.args.disableDelete ?? false;
+  get canDelete() {
+    return this.args.canDelete ?? false;
   }
 
   @tracked removeEditorModalOpen = false;
 
+  @tracked inviteTeamMemberModal = false;
+
   @action
   tryRemoveEditor() {
```

Comment:
Not sure if `tryRemoveEditor` is the most intuitive name for this -- maybe it could be `openRemoveEditorModal` or something similar?

Reply:
Agreed. If this opens the modal, there's a confirmation step before you try to remove the editor. Maybe `openRemoveEditorModal`?

Reply:
agree but it's not part of this diff